vms/platformvm/camino_service.go

```diff
@@ -408,6 +416,13 @@ func (s *CaminoService) Spend(_ *http.Request, args *SpendArgs, response *SpendR
 		}
 	}
 
+	bytes, err = txs.Codec.Marshal(txs.Version, owners)
+	if err != nil {
+		return fmt.Errorf("%w: %s", errSerializeOwners, err)
+	}
+	if response.Owners, err = formatting.Encode(args.Encoding, bytes); err != nil {
+		return fmt.Errorf("%w: %s", errSerializeOwners, err)
```

Comment:
Both are errors so should be `%w` (also please fix prev occurences)
```suggestion
		return fmt.Errorf("%w: %w", errSerializeOwners, err)
```

Reply:
Double %w not supported by golang

Reply:
Ouch! Thx